website/docs/podman/accessing-podman-from-another-wsl-instance.md

```diff
@@ -0,0 +1,235 @@
+   The Podman Machine creates this file with specific permissions.
+   To communicate with the Podman Machine from your WSL distribution your user must have write permissions for the socket.
+
+   To give access to the remote Podman machine to your user: create the group if necessary, assign group membership, and exit your session on the WSL distribution to apply the new group membership:
+
+   ```shell-session
+   $ sudo usermod --append --groups 10 $(whoami)
+   $ exit
+   ```
+
+## Testing the connection
+
+Verify that, on your WSL distribution, the Podman CLI communicates with your Podman machine.
+
+1. Start a session in your WSL distribution:
+
+   ```shell-session
+   > wsl
+   ```
+
+1. Verify that your user is member of the group delivering access to the remote Podman Machine socket:
+
+   ```shell-session
+   $ groups
+   ```
+
+   On the default Ubuntu WSL, the list contains the `uucp` group.
+
+   <details>
+   <summary>
+   On a custom WSL distribution, the group name might be different.
+
+   Find the required group name:
+   </summary>
+   <div>
+   The required group id is the same on any WSL distribution.
+
+   However, the group name might be different on a custom WSL distribution.
+
+   On the Podman Machine, which runs on a Fedora distribution:
+
+   - Rootful Podman: GID `10` name is `wheel`.
+   - Rootless Podman: GID `1000` name is `user`.
+
+   On the Ubuntu distribution:
+
+   - Rootful Podman: GID `10` name is `uucp`.
+   - Rootless Podman: GID `1000` name is the same as the user name you chose when creating the WSL machine.
+
+   On a custom WSL distribution, find the group name for:
+
+   - Rootful Podman:
+
+     ```shell-session
+     $ getent group 10
+     ```
+
+   - Rootless Podman:
+
+     ```shell-session
+     $ getent group 1000
+     ```
+
+   </div>
+   </details>
+
+1. Verify that Podman default system connections is set to your remote Podman machine:
+
+   ```shell-session
+   $ podman system connection list
+   ```
+
+1. Verify that Podman has a `Server` version corresponding to your Podman Machine version:
+
+   ```shell-session
+   $ podman version
+   ```
+
+   Sample output:
+
+   ```shell-session
+   Client:
+   Version:      3.4.4
+   API Version:  3.4.4
+   Go Version:   go1.18.1
+   Built:        Thu Jan 1 01:00:00 1970
+   OS/Arch:      linux/amd64
+
+   Server:
+   Version:      4.8.3
+   API Version:  4.8.3
+   Go Version:   go1.21.5
+   Built:        Wed Jan 3 15:11:40 2024
+   OS/Arch:      linux/amd64
+   ```
+
+   :::info
+
+   On your environment, the Podman version might be different.
+
+   :::
+
+1. Verify that you can list running containers.
+
+   [On the Podman Desktop graphical interface, start a container such as `quay.io/podman/hello`](/docs/containers/starting-a-container).
+
+   On your WSL distribution, list running containers:
+
+   ```shell-session
+   $ podman ps -a
+   ```
+
+   The output lists the same containers that you see on Podman Desktop graphical interface.
+
```

Comment:
In order to reinforce the behavior change coming from WSL/Docker I recommend finishing with some additional context:

e.g. insert a comment that podman desktop only has visibility to either rootless or rootful containers but not both at the same time. This can be switched with `podman machine set --rootful=false` and `podman machine set --rootful=true` and a corresponding change in the `podman system connection` configuration.